core/src/banking_stage/transaction_scheduler/scheduler_controller.rs

```diff
@@ -584,6 +584,14 @@ impl<T: LikeClusterInfo> SchedulerController<T> {
                     .zip(fee_budget_limits_vec.drain(..))
                     .zip(check_results)
                     .filter(|(_, check_result)| check_result.is_ok())
+                    .filter(|((((_, tx), _), _), _)| {
+                        Consumer::check_fee_payer_unlocked(
+                            &working_bank,
+                            tx.message(),
```

Comment:
In 2.1, this `Consumer::check_fee_payer_unlocked` takes a `&SanitizedMessage` instead of the trait reference. Adding `.message()` leads to multi-line formatting change from the original PR.